plugins/single_plugins/expo.cpp

```diff
@@ -71,8 +74,18 @@ class wayfire_expo : public wf::plugin_interface_t
     } state;
 
     int target_vx, target_vy;
+    int initial_vx, initial_vy;
```

Comment:
use `wf::point_t` in new code :) makes it easier to use if we need to pass the points around.

Reply:
Makes sense, I changed for both (didn't make sense to have separate x/y for target if initial workspace is wf::point_t)